File: Source/Fantasy.Jobs/Licence.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Fantasy.XSerialization;
using System.Xml.Linq;
using System.Reflection;

namespace Fantasy.Jobs
{
    [XSerializable("jobservice",NamespaceUri= Consts.LicenceNamespaceURI)]
    internal class Licence
    {

        [XAttribute("expire")]
        internal DateTime ExpireTime = DateTime.MinValue;

        [XAttribute("machine")]
        internal string MachineCode = string.Empty;

        [XAttribute("dev")]
        internal bool IsDevelopVersion = false;

        [XAttribute("slave")]
        internal int SlaveCount = 0;

    }
}

File: branches/Stheno/Fantasy.Jobs/src/fantasy/jobs/Licence.java
package fantasy.jobs;

import fantasy.xserialization.*;


@XSerializable(name = "jobservice", namespaceUri= Consts.LicenceNamespaceURI)
class Licence
{
	@XAttribute(name ="expire")
	public java.util.Date ExpireTime = new java.util.Date(Long.MIN_VALUE);


	@XAttribute(name = "machine")
	public String MachineCode = "";


	@XAttribute(name = "dev")
	public boolean IsDevelopVersion = false;


	@XAttribute(name = "slave")
	public int SlaveCount = 0;

	@XAttribute(name = "key")
	public String key = "";

}